src/goto-programs/remove_exceptions.cpp

```diff
@@ -7,7 +7,10 @@ Author: Cristina David
 Date:   December 2016
```

Comment:
Can you squash "Fixed bugs in remove_exceptions: " into previous commits if this is fixing bugs in the same PR?